pkg/server/routes.go

```diff
@@ -100,13 +102,28 @@ func (s *Server) routes(
 		}
 	}
 
-	var cedarLDAPClient cedarldap.Client
-	cedarLDAPClient = cedarldap.NewTranslatedClient(
-		s.Config.GetString(appconfig.CEDARAPIURL),
-		s.Config.GetString(appconfig.CEDARAPIKey),
-	)
+	//TODO: update this to have OKTA API live in it's own package?
+	var userSearchClient usersearch.Client
 	if s.environment.Local() || s.environment.Test() {
-		cedarLDAPClient = local.NewCedarLdapClient(s.logger)
+		userSearchClient = local.NewCedarLdapClient(s.logger)
+	} else {
+		useOKTAAPI := s.Config.GetBool(appconfig.USEOKTAAPI)
```

Comment:
Hey @DylanSpOddball! Lee, Clay and I had discussed this. We decided that it just made more sense to use an env var for the sake of simplicity. This isn't mean to be a long lived flag that is going to be toggled, but instead a temporary mechanism. Launch Darkly was going to be a larger lift, and the ability to switch live doesn't seem very important for this use case.

Reply:
@StevenWadeOddball Gotcha, that makes sense. Are we still going to do the full sequence of testing from the Jira ticket? (Deploy with `USE_OKTA_API=false`, check that everything works with the new code but still using LDAP, switch to using Okta and redeploy, test again?)

Reply:
Thanks @DylanSpOddball, I forget where we landed on that. I think it would probably make sense to do that validation in a lower environment at least to validate that the cedar code logic is still functional. Personally, I think it would be fine to do the final deploy with the `USE_OKTA_API=TRUE` if `USE_OKTA_API=false` is validated in a lower env. Thoughts? CC @ClayBenson94, @mynar7

Reply:
I agree with those thoughts, for what it's worth; maybe set `USE_OKTA_API=false` in dev, set `USE_OKTA_API=true` in impl, test in both to make sure both ways work. (We just would need to make sure we go back and set `USE_OKTA_API=true` in dev for future deploys, though)

Reply:
I think it's probably fine to cut straight to `USE_OKTA_API=true` in our envs, test in dev/impl, and if something goes wrong, we can always fall back on the config var, but I wouldn't want to have to run 6 tests (off/on, one for each env) for this when I think we can probably validate `USE_OKTA_API=true` in LLEs first

Reply:
I put a top-level comment on this PR and a comment in Jira describing the planned deployment process. @mynar7, if this all sounds good to you, you can resolve this thread.